Added basic docs (#280)
codatio/codat-docs: docs/assess/guides/migration-guides/deprecation-account-categories.md

---
title: "Migrate from V2 to V3 Enhanced Financials "
description: "Detailed instructions on how you can migrate from the features you're using today"
sidebar_label: "Migrating to the latest Enhanced Financials"
createdAt: "2022-10-06T08:32:16.232Z"
updatedAt: "2022-11-10T14:01:08.149Z"
---

## What is happening?

> On the **10th July 2023**, we will be deprecating the legacy version of Account Categories and all related features. These features include:

Reviewer: ```suggestion
On the **10th July 2023**, we will be deprecating the legacy version of Account Categories and all related features, including:

```

- Account categories
- Enhanced Balance Sheet (legacy version only)
- Enhanced Profit and Loss (legacy version only)
- Financial metrics
- Marketing metrics

To understand exactly what will be changing & how to leverage alternatives (including migration instructions) refer to the pages below:

- [Clients who have built to the Assess API](docs/assess/guides/underwriting/migration-guides/deprecation-account-categories-api-clients)

> - [Clients who use Assess in the portal](docs/assess/guides/underwriting/migration-guides/deprecation-account-categories-portal)

Reviewer: ```suggestion
- [Clients who use Assess in the Portal](docs/assess/guides/underwriting/migration-guides/deprecation-account-categories-portal)

```

> - [Clients who use the data extract to excel in the portal](docs/assess/guides/underwriting/migration-guides/deprecation-account-categories-v2-excel)

Reviewer: ```suggestion
- [Clients who use the data extract to Excel in the Portal](docs/assess/guides/underwriting/migration-guides/deprecation-account-categories-v2-excel)

```
